Main.java

```diff
@@ -9,7 +11,23 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+   // Creates a scanner used for input
+   Scanner input = new Scanner(System.in);
+
+   //declare and initialize a constant to store the first integer 
+  int F_INTEGER = 10;
+
+   //declare and initialize the a variable to store the second integer 
+   int sInteger = 3;
+
+   System.out.println("Please enter in two integers, on seperate lines, to divide.");
+
+      //declare and calculate the quotient and remainder 
+   int quot = F_INTEGER / sInteger;
+
+   //Tell the user the quotient and remainder
+   System.out.println("10/3 is " + sInteger + " with a remainder of 1");
```

Comment:
This is part of the question. You will also need to determine the remainder from division. This can be done by using the modulus operator (%) the same way you would use division.